content/en/localstack/lambda-executors.md

```diff
@@ -45,10 +45,29 @@ A typical invocation of a dummy python lambda can take around 3 seconds from sta
 All supported lambda types can be used with this executor.
 
 
-### Docker re-use
+### Docker reuse
 
 Configuration: `LAMBDA_EXECUTOR=docker-reuse`
 
+#### Stay-open mode
+LocalStack allows to use the stay-open mode of its lambda containers.
```

Comment:
nit: We're usually writing `Lambda` with upper-case L, to make it clear that we're referring to the service name (not critical at all, though..).